### PR TITLE
Improve UI/UX for Location Map page

### PR DESCRIPTION
**Pull request by azaria-dedmon**
**What does this PR do?**

- This PR adds navigation controls to map page.
- Improves UI/UX

**Description of task to be completed.**

- Adds user ability to zoom in and out of `map`
- Improve `navbar` on page

**How should this be manually tested?**

- Run `Python3 run.py`
- Go to` localhost:5000/`
- Select a state from the drop down list
- Click the `search` button
- Click the `plus` or `minus` controls on the map.

### CODE REVIEW

**Comment by alexkayabula** [comment]
@azaria-dedmon fix the search option in the resulting`navbar` accordingly.

**Comment by azaria-dedmon** [comment]
> @azaria-dedmon fix the search option in the resulting`navbar` accordingly.

Okay.

File: app/templates/location.html [hunk 1]
```diff
@@ -14,15 +14,19 @@
     <div class="nav">
       <div class="nav">
         <ul>
-            <input type="text" placeholder="Search users">
+          <form action="/search-user" method="get" id="search-form">
+            <input class="search" type="text" name="username" placeholder="Enter Username">
+            <input class="search-btn" type="submit" value="search">
```
Comment (alexkayabula) [inline app/templates/location.html]: @azaria-dedmon `search` should be capitalized. Also make sure the search button is exactly the same width as the search field.

**Review by alexkayabula — CHANGES_REQUESTED** [review]

**Review by alexkayabula — APPROVED** [review]
LGTM.Thank you.

**Pull request merged**